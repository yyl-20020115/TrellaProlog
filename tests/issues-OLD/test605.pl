main :-
	F=f(F),
	L1=[L1],
	write(L1), nl,
	L2=[1|L2],
	write(L2), nl,
	L3=[L3|1],
	write(L3), nl,
	L4=[L4|L4],
	write(L4), nl,
	true.

:- initialization(main).
