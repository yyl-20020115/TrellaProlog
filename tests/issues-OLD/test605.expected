[L1]
[1|L2]
[L3|1]
[L4|L4]
